components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -493,8 +493,7 @@ internal sealed class SubscriptionWrapper : ISubscription {
                     rawSubscription.ApplyChanges();
                     _currentlyMonitored = nowMonitored;
 
-                    var map = _currentlyMonitored.ToDictionary(
-                        k => k.Template.Id ?? k.Template.StartNodeId, v => v);
+                    var map = _currentlyMonitored.ToDictionary(k => k.Template.StartNodeId, v => v);
```

Comment:
Would the usage of this dictionary on line `499` be unaffected by this change ? That one uses `tem.Template.TriggerId` as search key.

Reply:
Yes, it's unaffected because triggering functionality is not used at all, it's not part of the API and was never tested. Actually I believe before the change was totally broken.